docker/.env.example

```diff
@@ -1,14 +1,19 @@
 
 # For configuration details, see: https://hawkins.gitbook.io/dispatch/configuration/app
+
 # General
 DISPATCH_DOMAIN=""
+DISPATCH_HELP_EMAIL = ""
+DISPATCH_HELP_SLACK_CHANNEL = ""
 
+# Persistence
 DATABASE_HOSTNAME=""
 DATABASE_CREDENTIALS=""
 
+# Authentication
 JWKS_URL=""
-DISPATCH_HELP_EMAIL = ""
-DISPATCH_HELP_SLACK_CHANNEL = ""
+VUE_APP_DISPATCH_OPEN_ID_CONNECT_URL=""
+VUE_APP_DISPATCH_CLIENT_ID=""
```

Comment:
These variables live in the `.env` file located in https://github.com/Netflix/dispatch/tree/develop/src/dispatch/static/dispatch. Would you mind creating a new `.env.example_vue_app` file and add them there? We may also want to add a note to let people know where the file lives.